concent_api/utils/api_view.py

```diff
@@ -38,7 +38,7 @@ def wrapper(request, *args, **kwargs):
         if request.content_type not in ['application/octet-stream', '', 'application/json']:
             return JsonResponse({'error': 'Concent supports only application/octet-stream and application/json.'}, status = 415)
 
-        if len(request.body) == 0:
+        if len(request.body) == 0 or request.body == b'{}':
```

Comment:
Why is this change back? I thought we resolved that earlier. `{}` is not an empty message.